### ISSUE TITLE
InputNumber: deleting field's content with the Delete key doesn't change the backing value

### ISSUE THREAD

**Issue by VsevolodGolovanov** [primefaces/primereact#1474]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

**Codesandbox Case (Bug Reports)**
https://codesandbox.io/s/inputnumberbug-zdb71

**Current behavior**
Deleting digits with the Delete key removes the digits from the displayed field, but does not affect the actual state. onChange doesn't get called.

**Expected behavior**
Delete key should affect the state, same as Backspace. onChange should be called.

**Minimal reproduction of the problem with instructions**
Open the linked codesandbox. Try deleting digits with Delete. See the disrepancy:
![image](https://user-images.githubusercontent.com/8070317/88943696-7a798b00-d294-11ea-8754-1a52e5368cd9.png)

**Please tell us about your environment:**

* **React version:**

* **PrimeReact version:**
4.2.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by suatalim** [comment]
Same problem.
"del" key not updating value.

**Comment by mertsincan** [comment]
Fixed for 5.0.

Best Regards,
